packages/common/src/search/index.ts

```diff
@@ -7,3 +7,4 @@ export * from "./types";
 export * from "./_searchContent";
 export * from "./_searchGroups";
 export * from "./_searchUsers";
+export * from "./searchContentEntities";
```

Comment:
Does this need to be exported, seems like it's just for our internal use, right?